### PR TITLE
feat: OGL case licence

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: cases/views/main.py [hunk 1]
```diff
@@ -415,6 +417,33 @@ def post(self, request, **kwargs):
...
+    def post(self, request, **kwargs):
+        self.init(request, **kwargs)
+        if not request.POST.get("confirm"):
+            return form_page(
+                request,
+                self.get_form(),
+                data=self.get_data(),
+                errors={"confirm": [Manage.ReissueOGEL.ERROR]},
+                extra_data=self.context,
+            )
```
Comment (jfaracik) [inline cases/views/main.py:440]: Checking the post data would be better suited inside of the service or inside of a validator :)

File: ui_automation_tests/pages/application_page.py [hunk 2]
```diff
@@ -285,3 +285,6 @@ def go_to_cases_activity_tab_for_eua(self, internal_url, context):
             + context.eua_id
             + "/activity/"
         )
+
+    def click_activity_tab(self):
+        self.driver.find_element_by_id(self.CASE_NOTES_AND_ACTIVITY_TAB).click()
```
[the remark this answers is not in the document]
Comment (BenAndrew-Kainos) [reply ui_automation_tests/pages/application_page.py:290]: Requires the case_pk which we don't have for OGL's (annoyingly), so this is the simplest solution.

File: cases/views/main.py (hunk 1 above)
Comment (jfaracik) [reply cases/views/main.py:440]: That's fine :star:

File: ui_automation_tests/pages/application_page.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (BenAndrew-Kainos) [reply ui_automation_tests/pages/application_page.py]: Sorry I misread your comment. Fixed now

**Pull request merged**